man/systemd.netdev.xml

```diff
@@ -691,6 +691,15 @@
           The valid range is 0-1048575.
           </para>
         </listitem>
+    </varlistentry>
+      <varlistentry>
+        <term><varname>IPDoNotFragment=</varname></term>
+        <listitem>
+          <para>Allows to set the IPv4 Do not Fragment (DF) bit in outgoing packets,
+          or to inherit its value from the IPv4 inner header. If the encapsulated protocol
+          is IPv6 and DF is configured to be inherited, always set it. Takes a boolean value,
```

Comment:
> If the encapsulated protocol is IPv6 and DF is configured to be inherited, always set it.

Always set what? Could you elaborate more?

Reply:
It means that If VXLAN goes via IPv6 then set to inherit always else it' wont work.

          <para>Allows to set the IPv4 Do not Fragment (DF) bit in outgoing packets,                                                                                                          
          or to inherit its value from the IPv4 inner header. Set to <literal>inherit</literal>                                                                                               
          if the encapsulated protocol is IPv6. Takes a boolean value, or <literal>inherit</literal>.                                                                                         
          When unset, the kernel's default will be used.</para>

Reply:
Thanks.